src/hotspot/share/opto/c2compiler.cpp

```diff
@@ -225,6 +229,12 @@ bool C2Compiler::is_intrinsic_supported(const methodHandle& method) {
   case vmIntrinsics::_copyMemory:
     if (StubRoutines::unsafe_arraycopy() == nullptr) return false;
     break;
+  case vmIntrinsics::_electronicCodeBook_encryptAESCrypt:
+    if (StubRoutines::electronicCodeBook_encryptAESCrypt() == nullptr) return false;
+    break;
+  case vmIntrinsics::_electronicCodeBook_decryptAESCrypt:
+    if (StubRoutines::electronicCodeBook_decryptAESCrypt() == nullptr) return false;
+    break;
```

Comment:
These two intrinsics were marked as supported on non-x86 platforms where the underlying stubs are not generated

Reply:
good catch